docs/rfcs/README.md

```diff
@@ -1,10 +1,75 @@
+This folder contains accepted design documents for the Collector. Proposals here imply changes only
+on the OpenTelemetry Collector and not on other parts of OpenTelemetry; if you have a cross-cutting
+proposal, file an [OTEP][1] instead.
 
-Accepted RFCs:
+# RFC Process
 
-- [OpenTelemetry Collector Processor Exploration](./processing.md)
+## Scope
+
+The Request For Comments (RFC) process is intended to be used for significant changes to the Collector. Major design
+decisions, specially those that imply a change that is hard to reverse, should generally be
+documented as an RFC. Controversial changes should also be documented as an RFC, so that the
+community can have a chance to provide feedback. The goal of this process is to ensure we have a
+coherent vision before embarking on significant work.
+
+Ultimately, if any opentelemetry-collector maintainer feels that a change requires an RFC, then a
+merged RFC is a requirement for said change.
+
+## Contents
+
+We are purposefully light on the structure of RFCs, with the focus being the decision process and
+not the document itself. When in doubt, the [OTEP template][2] can be a good starting point.
+Regardless of the structure, the RFC should make clear what commitments are being made by the
+Collector SIG when merging it.
+
+## Announcement
+
+RFCs should be announced in a Collector SIG meeting and on the #otel-collector-dev Slack channel.
+
+## Approval process
+
+An RFC is just like any other PR in this repository, with the exception of more stringent criteria
+for merging it.
+
+### Stakeholders
+
+To define merge criteria and voting, each RFC has a set of 'stakeholders'. All
+opentelemetry-collector approvers are considered stakeholders. Additional stakeholders (e.g.
+maintainers of related SIGs or experts) may be explicitly noted in the RFC.
+
+### Merge rules
+
+We use a Lazy Consensus method with the following rules:
+
+1. *Quorum*: For an RFC to be mergeable, it needs to have at least **two approvals** from the
+   approvers set as well as approvals from any additional stakeholders.
+2. *Waiting period*: Maintainers need to announce their intent to merge the RFC with a GitHub
+   comment. They will need to add a `rfc:final-comment-period` label to the PR and wait for at least
+   **4 business days** after making the announcement to merge the RFC.
+3. *Objections*: Objections should be communicated as a 'request changes' review. All objections
+   must be addressed or voted before merging an RFC. If unable to resolve an objection, any
+   maintainer may call for a vote to be made on the objection (see below). This will be signified by
+   adding a `rfc:vote-needed` label to the PR. The voting result is binding and a maintainer can
+   drop any 'request changes' based on the vote results or consensus.
+4. *Modifications*: Non-trivial modifications to an RFC reset the waiting period. RFC authors must
+   re-request reviews if the RFC has been modified significantly.
```

Comment:
```suggestion
   re-request any approving, comment, or 'request changes' reviews if the RFC has been modified significantly.
```
My understanding is that we don't want to have approving reviews applied to an outdated version of the RFC, so I think we should be explicit on that here. For the sake of completeness, we can probably just enumerate the other types of reviews GitHub supports re-requesting a review on as well.

Reply:
Makes sense, added!